addons/payment/models/account_payment_method.py

```diff
@@ -19,8 +19,9 @@ class AccountPaymentMethodLine(models.Model):
 
     @api.depends('payment_method_id')
     def _compute_payment_acquirer_id(self):
+        code = ['transfer' if c == 'manual' else c for c in self.mapped('code')]
```

Comment:
you could reduce this to
`self.mapped(lambda pm: 'transfer' if pm.code == 'manual' else pm.code)`